README.md

```diff
@@ -12,3 +12,8 @@ Here is some important information that should be included in a README for an op
 - [ ] link to the open source license
 - [ ] link to community Discord, Slack group, or GitHub Discussions
 
+## Helpful links:
+[Contributing md](https://github.com/eram23/progress-tracker.git)
+[Code of conduct](https://github.com/scrimba/community/blob/master/CODE-OF-CONDUCT.md)
```

Comment:
We don't have a Code Of Conduct in this repository. There is a separate issue for that. Once it is created, we can add a link here.

Reply:
Gotcha, will leave the URL empty for now then if that is ok

Reply:
Yeah, that's fine!